README.md

```diff
@@ -93,10 +93,26 @@ Note: All example values are the actual values for Ethereum Mainnet and the stag
 
 Run unit tests on the local Hardhat network:
 
+To test DssVestTopUp.sol:
+
+```bash
+npm run test:topup
+```
+
+To test DssCronKeeper.sol:
+
 ```bash
-npm run test
+npm run test:keeper
 ```
 
+For end-to-end testing:
+
+```bash
+FORK_URL="https://eth-goerli.g.alchemy.com/v2/<YOUR ALCHEMY KEY>" BLOCK_NUMBER="1234" npm run test:e2e
```

Comment:
Let's use the goerli rpc url from env (since it's intended to be tested with it) and also put the for block number there. This way we hide these details from the user.